### ISSUE TITLE
'unable to verify the first certificate' error when sending over HTTPS

### ISSUE THREAD

**Issue by pi-kei** [artsoroka/bytehand#1]
Do you get 'unable to verify the first certificate' error when sending over HTTPS?

**Comment by artsoroka** [comment]
@pi-kei Yes, when using HTTPS you might see UNABLE_TO_VERIFY_LEAF_SIGNATURE error. Quickfix: turn of unauthorized certs rejection like so 

`process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = '0'` 

Can you specify node version you're using?

**Comment by pi-kei** [comment]
@artsoroka I'm using node v6.10.3.
According to this page https://www.npmjs.com/package/ssl-root-cas error occurs when `the webserver you are connecting to is misconfigured and did not include the intermediate certificates in the certificate it sent to you`. I guess this means that bytehand's API webserver is miscofigured and it can be fixed only on their side.

**Comment by pi-kei** [comment]
@artsoroka Problem with certificates solved but new problem came out. When sending over HTTPS cyrillic symbols are messed up.
To add missing certificates (works on windows and chrome):

1. Open browser and put into address line `https://bytehand.com:8443/send` and make request. Open dev tools -> Security -> View certificate
2. Export all intermediate certificates as cer in der
3. Using `openssl` convert cer files to pem `openssl x509 -in cert.crt -inform der -outform pem -out cert.pem`
4. Use this package to add those pem-files https://www.npmjs.com/package/ssl-root-cas

**Comment by pi-kei** [comment]
@artsoroka If use `/send_multi` then cyrillic symbols are ok.

**Comment by artsoroka** [comment]
@pi-kei bytehand support only GET request for sending single message. And while it works just fine for plain requests, with SSL enabled, message body seems to be corrupted 

In order to fix that, `send_multi` should be used both for single and multiple messages. Fix that soon. Thanx for the tip
